lib/contracts/code_generator.js

```diff
@@ -363,16 +365,16 @@ class CodeGenerator {
       function getImports(web3Location, next) {
         web3Location = web3Location.replace(/\\/g, '/'); // Import paths must always have forward slashes
         code += "\nimport Web3 from '" + web3Location + "';\n";
+        code += "\nglobal.Web3 = Web3;\n";
```

Comment:
This line is useless as you already do it after the if

Reply:
I think it's a case of capital-W `Web3` vs. `web3`, if you're referring to [this if](https://github.com/embark-framework/embark/blob/fdd813c5ce3ab90807fd46d3f008bfaa863f6866/lib/contracts/code_generator.js#L370).